website/docs/d/network.html.markdown

```diff
@@ -43,6 +60,9 @@ The following arguments are supported:
   network objects, the ID of the distributed virtual switch for which the port
   group belongs. It is useful to differentiate port groups with same name using
   the distributed virtual switch ID.
+* `additional_filters` - (Optional) For standard virtual switch port groups, distributed virtual switch port groups and NSX port groups.
+  Networks are found by setting `network_type` to Network, DistributedVirtualPortgroup or OpaqueNetwork. This is required if you have
```

Comment:
Maybe have this as a supported filters section?

Reply:
removing filters